platform/src/main/resources/default-config.yaml

```diff
@@ -1,47 +1,53 @@
 stellar:
   anchor:
-    config: in-memory                             # Configure the application from this file after it is loaded in memory
+    # Configure the application from this file after it is loaded in memory
+    # If `config` is in-memory, the yaml file contains all settings for the server.
+    config: in-memory
     app-config:
       type: config-spring-property                # Activate [config-spring-property] module
-      settings: config-spring-property-settings   # The location of the configuration data in this file.
+      settings: app-config                        # The location of the configuration data
     data-access:
       type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
-      settings: data-spring-jdbc-settings         # The location of the configuration data in this file.
+      settings: data-spring-jdbc-sqlite           # The location of the configuration data in this file.
     logging:
       type: logging-logback
       settings: logging-logback-settings
-    payment-gateway:
-      services:
-        - payment-circle                  # Activate [payment-circle] module
-        - payment-stellar                # Activate [payment-stellar] module
 
 #
 # Application settings
 #
-config-spring-property-settings:
+app-config:
   # general
   app:
-    stellarNetworkPassPhrase: Test SDF Network ; September 2015
+    stellarNetworkPassphrase: Test SDF Network ; September 2015
     hostUrl: http://localhost:8080
     languages: en
-    horizonURI: https://horizon-testnet.stellar.org
+    horizonUrl: https://horizon-testnet.stellar.org
     assets: assets-test.json
     jwtSecretKey: secret
 
   # sep-1
   sep1:
     enabled: true
-    stellarFile: sep1/stellar-wks.toml
+    #
+    # The stellarFile is the location where to retrieve the content as the return value
+    # of the `/.well-known/stellar.toml`.
+    #
+    # The format of the resource is specified as Table 6.1 of the Spring document located at:
+    # https://docs.spring.io/spring-framework/docs/3.2.x/spring-framework-reference/html/resources.html
+    #
+    stellarFile: classpath:sep1/stellar-wks.toml
 
   # sep-10
   sep10:
     enabled: true
     homeDomain: localhost:8080
     clientAttributionRequired: false
-    clientAttributionAllowList: lobstr.co,preview.lobstr.co
+    clientAttributionAllowList: lobstr.co,preview.lobstr.co # use this if we want to white list
+    # clientAttributionDenyList: # use this if we want to black list.
     authTimeout: 900
     jwtTimeout: 86400
-    signingSeed:
+    signingSeed: SAX3AH622R2XT6DXWWSRIDCMMUCCMATBZ5U6XKJWDO7M2EJUBFC3AW5X
```

Comment:
We shouldn't provide a secret key here should we?

Reply:
This is already being used in other yaml files. We still have to figure out how to inject this using Kubernetes secrets.

Reply:
We have this TODO in kube the k8s repo: https://github.com/stellar/kube/blob/fec12a77e08c911d9799bc042414b891ea1076a3/kube001-dev/namespaces/common-previews/anchor-platform-dev/sep-server-config-map.yaml#L9